acs-integration-tests/pom.xml

```diff
@@ -509,7 +509,7 @@
                         <artifactId>maven-failsafe-plugin</artifactId>
                         <configuration>
                             <forkCount>0</forkCount>
-                            <parallel>false</parallel>
+                            <parallel>methods</parallel>
```

Comment:
should add a comment here in the pom.xml.

This setting does not expect a boolean (see http://maven.apache.org/surefire/maven-failsafe-plugin/examples/fork-options-and-parallel-execution.html).   Currently all test methods are running concurrently which causes failures in ACSMeteringIT  (see https://rally1.rallydev.com/#/59288076735ud/detail/defect/59347373065).  

'methods' implies that only test methods within a test class are executed concurrently.